pep-9999.rst

```diff
@@ -0,0 +1,883 @@
+intuitive and more structured way to express the license of a distribution using
+a well defined syntax and well known license ids.
+
+Over time, recommending the usage of these expressions will help Python package
+publishers improve the clarity of their license documentation to the benefit of
+packages authors, consumers and redistributors.
+
+
+Core Metadata Specification updates
+===================================
+
+The canonical source for the names and semantics of each of the supported
+metadata fields is the Core Metadata Specification [#cms]_ document.
+
+The details of the updates considered to the Core Metadata Specification [#cms]_
+document as part of his PEP are detailed here and will be added to the
+canonical source once this PEP is approved.
+
+
+Added in Version 2.2
+--------------------
+
+License-File (multiple use)
+:::::::::::::::::::::::::::
+
+The is a string that is a package-root relative path to a license file. The
+license file content __must__ be UTF-8-encoded text.
+
+Build tools should honor this field and include the corresponding license
+file(s) in the built package.
+
+
+Changed in Version 2.2
+----------------------
+
+License (optional)
+::::::::::::::::::
+
+Text indicating the license covering the distribution. This text can be either a
+valid `License Expression` as defined here or any free text.
+
+Publishing tools SHOULD issue an informational warning if this field is empty or
+missing or is not a valid `License Expression` as defined here. Build tools MAY
+issue such a warning too.
+
+
+License Expression syntax
+'''''''''''''''''''''''''
+
+A `License Expression` is a string using the SPDX license expression syntax as
+documented in the SPDX specification [#spdx]_ using either Version 2.1
+[#spdx21]_ or a later compatible version. SPDX is a working group at the Linux
+Foundation that defines a standard way to exchange package information.
+
+When used in the `License` field and as a specialization of the SPDX license
+expression definition, a `License Expression` can use the following license
+identifiers:
+
+- any SPDX-listed license short-form identifiers that are published in the 
+  SPDX License List [#spdxlist]_ using either Version 3.6 of this list or any
+  later compatible version. Note that the SPDX working group never removes any
+  license identifiers: instead they may only one as obsolete.
+
+- the `Public-Domain` and `Proprietary` strings to support generic ids that are
+  not available in the SPDX license list.
+
+When processing the `License` field to determine if it contains a valid license
+expression, tools:
+
+- MUST ignore the case of the `License` field.
+
+- SHOULD report an informational warning if field does not contain a license
+  expression or if the license expression syntax is invalid or the license
+  identifiers are unknown or the license identifiers have been marked as
+  obsolete in the SPDX License List [#spdxlist]_.
```

Comment:
deprecated, not obsolete.

Reply:
corrected in 366481a919cf7fd875a47a875c6938bbdc6334f6
Thank you!

Reply:
Cheers.